apmpackage/apm/data_stream/app_metrics/manifest.yml

```diff
@@ -30,8 +30,10 @@ elasticsearch:
         - double:
             mapping:
               type: double
+              index: false
         - numeric_labels:
             path_match: numeric_labels.*
             mapping:
               type: scaled_float
+              index: false
```

Comment:
I don't think we want this. Labels may be used for searching; they're not necessarily metrics.

Reply:
Oh, good point! Updated :+1: